build-source-tarball.sh

```diff
@@ -233,6 +233,9 @@ done
 echo 'Copying source-built packages to tarball to replace packages needed before they are built...'
 mkdir -p $TARBALL_ROOT/packages/source-built
 cp -r $SCRIPT_ROOT/Tools/source-built/coreclr-tools $TARBALL_ROOT/packages/source-built/
+cp $SCRIPT_ROOT/bin/obj/x64/Release/blob-feed/packages/*Arcade*.nupkg $TARBALL_ROOT/packages/source-built/
+cp $SCRIPT_ROOT/bin/obj/x64/Release/blob-feed/packages/*SourceLink*.nupkg $TARBALL_ROOT/packages/source-built/
+cp $SCRIPT_ROOT/bin/obj/x64/Release/blob-feed/packages/*Build*Tasks*.nupkg $TARBALL_ROOT/packages/source-built/
```

Comment:
These should come from the source-built artifacts tarball.  If we need a version update, maybe this needs my change to update these to 3.0.100 version of that?  My PR is failing for that on building arcade in the tarball.  I'm wondering if I need these Arcade updates?

Reply:
I updated the source-built artifacts tarball but was still running into issues with the version numbers here.  Logged https://github.com/dotnet/source-build/issues/1321 to fix this.